### DISCUSSION TITLE
Assistance Needed with Pasting Chat-GPT Feedback into GPT-engineer for Clarification

### DISCUSSION THREAD

**Discussion by lrn2codenow** [AntonOsika/gpt-engineer#474] [Q&A]
I'm seeking your assistance in resolving an issue I'm encountering when pasting feedback from Chat-GPT 4 into GPT-engineer for clarification purposes. Currently, after obtaining appropriate clarifications from Chat-GPT 4, I face difficulties when copying and pasting the feedback into the GPT-engineer terminal. The code doesn't start properly, and I am prompted multiple times to confirm if the code ran successfully. I'm reaching out to the community for guidance in addressing this problem.

Background:
I have been using Chat-GPT 4 to clarify areas that gpt-engineer is asking for. Chat-GPT 4 provides valuable feedback to enhance the clarity of the code. However, when I attempt to copy and paste the provided feedback into the GPT-engineer terminal, I encounter issues that prevent the code from executing correctly. It appears to evaluate the feedback which sometimes is 60 lines of text and code. If this would work it would be super helpful. I am blown away by gpt-engineer. Truly amazing work. Clearly the future.

**Comment by Luckysin13** [comment]
Paste into a text editor and remove ALL carriage returns and line feeds.

Notepad++
Menu, View, Show Symbol, Show All Characters
